vars/params.env

```diff
@@ -1,5 +1,7 @@
-email=admin@onepanel.io
-commonName=*.test-cluster-5.onepanel.io
-
-aws_region=us-west-2
-aws_access_key=aws_access_key
\ No newline at end of file
+LOGGING_ELASTIC_SEARCH_IMAGE=docker.elastic.co/elasticsearch/elasticsearch:7.2.0
```

Comment:
@aleksandrmelnikov @Vafilor still a bit confused about the convention here, why all caps here and in other places we have all lower case? Shouldn't we follow YAML convention and use camelCase across the board or is there a limitation I'm not aware of?

Reply:
Also the generate param file ended up like snake case too:
```
argo:
  access_key: fillmeout
  artifact_repository_access_key_secret_key: ARGO_ACCESS_KEY
  artifact_repository_access_key_secret_name: mlpipeline-minio-artifact
  artifact_repository_endpoint: minio-service.onepanel:9000
  artifact_repository_secret_key_secret_key: ARGO_SECRET_KEY
  artifact_repository_secret_key_secret_name: mlpipeline-minio-artifact
  secret_key: fillmeout
logging:
  elastic_search_image: docker.elastic.co/elasticsearch/elasticsearch:7.2.0
  volume_storage: 1Gi
```

Reply:
another note on the generated file, we'll be using this object storage for everything so I think for we don't to put it specifically under `argo` but under `artifactRepository`. Thoughts?

Reply:
also, why do we have to secret names? shouldn't these all come from one secret name?

Reply:
If we're using this object storage for everything, not just argo, maybe we should go one step 
```
object_storage:
    argo:
       artifact_repository:
           <fields>
```

The multiple secret names are from Argo config. Their defaults do point to one secret name, but they allow the secret values to be across multiple names it looks like.

Reply:
Simpler view?
```
artifactRepository:
  s3:
     endpoint:
     bucket:
     accessKey:
     secretKey: 
```

Reply:
@aleksandrmelnikov @Vafilor not sure if we're using secrets anywhere else other than `istio-system/onepanel-tls` (which should only be dedicated to TLS certs) but if we are and it is storing credentials for external systems, we should also use that secret for this as well. My guess is that this secret will be under the `onepanel` namespace since that where the API, UI and argo workflows will be.